msteams-platform/bots/how-to/conversations/conversation-messages.md

```diff
@@ -439,20 +439,37 @@ Form completion message appears in Adaptive Cards while sending a response to th
+## Status codes from bot APIs
+
+Ensure to handle these errors appropriately in your Teams app. The following table lists the error codes and the descriptions under which the errors are generated:
+
+| Status code | Error code and message values | Description | Retry request | Developer action |
+|----------------|-----------------|-----------------|----------------|----------------|
+| 400 | **Code**: `Bad Argument` <br/> **Message**: *scenario specific | Invalid request payload provided by the bot. See error message for specific details. | No | Re-evaluate request payload for errors. Check returned error message for details. |
+| 401 | **Code**: `BotNotRegistered` <br/> **Message**: No registration found for this bot. | The registration for this bot wasn't found. | No | Verify the bot ID and password. Ensure that the bot ID is registered with Microsoft through bot Channel registration in Azure or Teams Developer Portal.|
+| 403 | **Code**: `BotDisabledByAdmin` <br/> **Message**: The tenant admin disabled this bot | Admin tenant has blocked the interactions between user and the bot app. Admin tenant needs to allow the app for the user. For more information, see [app policies](/microsoftteams/app-policies). | No | Stop attempting to post to conversation until interaction with bot is explicitly initiated by a user in the conversation, indicating that the bot is no longer blocked. |
+| 403 | **Code**: `BotNotInConversationRoster` <br/> **Message**: The bot isn't part of the conversation roster. | The bot isn't part of the conversation. Re-install the app in conversation. | No | Before attempting to send conversation requests, await for [`installationUpdate`](~/bots/how-to/conversations/subscribe-to-conversation-events.md#install-update-event) event, which indicates that the bot has been re-added.|
+| 403 | **Code**: `ConversationBlockedByUser` <br/> **Message**: User blocked the conversation with the bot. | User has blocked the bot in personal chat or a channel through moderation settings. | No | Delete the conversation from cache. Stop attempting to post to conversations until interaction with bot is explicitly initiated by a user in the conversation, indicating that the bot is no longer blocked. |
+| 403 | **Code**: `NotEnoughPermissions` <br/> **Message**: *scenario specific | Bot doesn't have required permissions to perform the requested action. | No | Determine the required action from the error message. |
+| 404 | **Code**: `ActivityNotFoundInConversation` <br/> **Message**: Conversation not found. | The message ID provided couldn't be found in the conversation. Message doesn't exist or it has been deleted. | No | Check if message ID is expected value and remove it from the cache. |
+| 404 | **Code**: `ConversationNotFound` <br/> **Message**: Conversation not found. | Conversation wasn't found as it doesn't exist or has been deleted. | No | Check if the value of conversation ID sent is as expected and delete conversation ID if cached. |
```

Comment:
`Check if the value of conversation ID sent is as expected and delete conversation ID if cached.
`
Please update to:
`Check if conversation ID sent is an expected value. Remove the id if it was cached.`